Dockerfile

```diff
@@ -1,49 +1,80 @@
-# To run: docker run -v /path/to/wsgi.py:/var/www/indexd/wsgi.py --name=indexd -p 81:80 indexd
-# To check running container: docker exec -it indexd /bin/bash
+ARG AZLINUX_BASE_VERSION=master
 
-FROM quay.io/cdis/python:python3.9-buster-2.0.0
+# Base stage with python-build-base
+FROM quay.io/cdis/python-build-base:${AZLINUX_BASE_VERSION} AS base
+
+# Comment this in, and comment out the line above, if quay is down
+# FROM 707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/python-build-base:${AZLINUX_BASE_VERSION} as base
```

Comment:
is there a reason we're not preferring ECR?

Reply:
So that community may be able to build/ develop and I belive ECR has a bandwidth cost, while quay is free.

Reply:
ahh, that makes sense 👍

Reply:
is that acceptable for our own internal usage? Or do we need some way to automate flipping that to ECR as part of build?